openmc/mesh.py

```diff
@@ -1052,6 +1052,7 @@ def __init__(self, mesh_id=None, name=''):
         self._r_grid = None
         self._phi_grid = [0.0, 2*pi]
         self._z_grid = None
+        self._origin = (0., 0., 0.)
```

Comment:
Since we're passing a valid value for the origin, we can use the setter here to ensure that they type is converted to a NumPy array. This is what was causing the tests to fail.
```suggestion
        self.origin = (0., 0., 0.)
```